stylesheets/_component.toggle-switch.scss

```diff
@@ -85,20 +85,15 @@
         color: color(white);
         content: '\F00C \F00D';
         font-family: 'FontAwesome';
-        font-size: 14px;
+        font-size: 16px;
         left: 10px;
-        letter-spacing: 10px;
+        letter-spacing: 14.4px;
```

Comment:
add magic number explanation in comment?

Reply:
Added